versioneer.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
was this file actually changed or was it all just autoformatting stuff?

Reply:
Running black locally changed it.